content/pt-br/docs/concepts/configuration/windows-resource-management.md

```diff
@@ -0,0 +1,75 @@
+content_type: concept
+weight: 75
+---
+
+<!-- overview -->
+
+Esta página descreve as diferenças em como os recursos são gerenciados entre o Linux e o Windows.
+
+<!-- body -->
+
+Em nós do Linux, {{< glossary_tooltip text="cgroups" term_id="cgroup" >}} são usados como um limite de pod para controle de recursos. 
+Os contêineres são criados dentro desse limite para o isolamento de rede, processo e sistema de arquivos. 
+As APIs Linux cgroup podem ser usadas para coletar estatísticas de uso de CPU, E/S e memória.
+
+Em contraste, o Windows usa um [_objetos de trabalho_](https://docs.microsoft.com/windows/win32/procthread/job-objects) por contêiner com um filtro de namespace do sistema
+para conter todos os processos em um contêiner e fornecer isolamento lógico ao hospedar.
+(Os objetos de trabalho são um mecanismo de isolamento de processo do Windows e são diferentes dos
+que o Kubernetes chama de {{< glossary_tooltip term_id="job" text="Job" >}}).
+
+Não há como executar um contêiner do Windows sem a filtragem de namespace. 
+Isso significa que os privilégios do sistema não podem ser declarados no contexto do host e, 
+portanto, os contêineres privilegiados não estão disponíveis no Windows.
+Os contêineres não podem assumir uma identidade do host porque o Gerente de conta de segurança (SAM) é separado.
+
+## Gerenciamento de memória {#resource-management-memory}
+
+O Windows não possui um eliminador de processo de falta de memória como o Linux. 
+O Windows sempre trata todas as alocações de memória do modo de usuário como 
+virtuais e os arquivos de paginação são obrigatórios.
+
+Os nós do Windows não sobrecarregam a memória para os processos. O efeito líquido 
+é que o Windows não atingirá as condições de falta de memória 
+da mesma forma que o Linux, e processará a página em disco em vez de estar 
+sujeito ao encerramento por falta de memória (OOM). Se a memória for 
+superprovisionada e toda a memória física estiver esgotada, a paginação poderá diminuir o desempenho.
+
+## Gerenciamento de CPU {#resource-management-cpu}
+
+O Windows pode limitar a quantidade de tempo de CPU alocado para diferentes processos, 
+mas não pode garantir uma quantidade mínima de tempo de CPU.
+
+No Windows, o kubelet oferece suporte a uma flag de linha de comando para definir a
+[prioridade do escalonador](https://docs.microsoft.com/windows/win32/procthread/scheduling-priorities) do processo kubelet:
+ `--windows-priorityclass`. Essa flag permite que o processo kubelet obtenha
+mais fatias de tempo de CPU quando comparado a outros processos em execução no host do Windows.
+Mais informações sobre os valores permitidos e os seus significados estão disponíveis em
+[Classes prioritárias do Windows](https://docs.microsoft.com/en-us/windows/win32/procthread/scheduling-priorities#priority-class).
+Para garantir que os Pods em execução não prejudiquem o kubelet de ciclos de CPU, defina essa flag como `ABOVE_NORMAL_PRIORITY_CLASS` ou acima.
```

Comment:
Some suggestions here:
* In the first sentence, use "classes de prioridade do Windows" instead of "Classes prioritárias do Windows".
* How would you feel about using "não deixem o kubelet sem ciclos de CPU", instead of "não prejudiquem o kubelet de ciclos de CPU"?